actors/runtime/indices/indices.go

```diff
@@ -146,25 +133,26 @@ func (inds *IndicesImpl) SectorWeight(
 ) abi.SectorWeight {
 	// for every sector, given its size, start, end, and deals within the sector
 	// assign sector power for the duration of its lifetime
-	PARAM_FINISH()
-	panic("")
+	// PARAM_FINISH()
+	sectorDuration := endEpoch - startEpoch
+	return abi.SectorWeight(big.NewInt(int64(sectorDuration) * int64(sectorSize)))
```

Comment:
This is the wrong units. Just return sectorSize if you want to ignore deal weight for now.

Reply:
Understood, it is unresolved how SectorWeight should be computed and integral of space over time can be an option. Will use sectorSize for now.